### ISSUE TITLE
[상품등록] 옵션추가

### ISSUE THREAD

**Issue by PLS012** [akasima/xero_commerce#140]
1. 상품등록시 "옵션추가"가 없고, "상품수정"을 다시 해야 옵션추가가 나옵니다.
-> 상품등록시 옵션추가를 바로 할 수 있도록 하면 좋겠습니다.

![상품등록시 옵션 추가가 없음](https://user-images.githubusercontent.com/64255511/80354846-ddc8db00-88a9-11ea-8c92-12f889e9a592.jpg)

------------------------------------------

2. 옵션추가에 데이터 입력시, 입력값을 넣지 않고 저장을 누르면 internal server error 가 납니다.

![옵션 추가시 데이터 입력없이 저장하면 에러](https://user-images.githubusercontent.com/64255511/80355560-d3f3a780-88aa-11ea-8a76-a9dff01c8165.jpg)
![옵션 추가시 데이터 입력없이 저장하면 에러 - internal server error](https://user-images.githubusercontent.com/64255511/80355575-d7872e80-88aa-11ea-88f6-a9f8f19d293a.jpg)

아래는 에러로그입니다.

production ERROR SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'name' cannot be null (SQL: insert into `xe_xero_commerce_product_option_item` (`product_id`, `option_type`, `name`, `addition_price`, `stock`, `alert_stock`, `state_display`, `state_deal`, `updated_at`, `created_at`) values (1, 2, , , , , 1, 1, 2020-04-27 18:14:43, 2020-04-27 18:14:43)) {"userId":"2cd54ce7-bfa3-4a08-a681-4b918bcac5a1","email":"simplenclear@gmail.com","exception":"[object] (Illuminate\\Database\\QueryException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'name' cannot be null (SQL: insert into `xe_xero_commerce_product_option_item` (`product_id`, `option_type`, `name`, `addition_price`, `stock`, `alert_stock`, `state_display`, `state_deal`, `updated_at`, `created_at`) values (1, 2, , , , , 1, 1, 2020-04-27 18:14:43, 2020-04-27 18:14:43)) at /home/pls/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:664, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'name' cannot be null at /home/pls/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php:458)

**Comment by darron1217** [comment]
에러는 옵션명을 넣지 않아서 발생하는 문제인것 같네요...
옵션명을 넣어주면 될것 같습니다

**Comment by PLS012** [comment]
네. 2번 문제에서, 옵션명을 넣지 않아서 그렇죠. 그런데, 넣던 넣지 않던 있는 그대로 저장이 되어야 하는게 맞는 것 같습니다. 아니면, '옵션명을 넣어주세요.' 라는 알람 메시지가 뜬다든가, 빨간색 글씨로 에러 내용을 각 데이터 입력창 아래에 나오게 한다든가 하는게 좋을 것 같습니다.
제가 개발자가 아니라서 어떻게 할 수 있는게 없습니다. 
그래서 XE 대표님께서 빨리 복귀하셔서 업그레이드 시켜 달라고 Issue 에 잔뜩 달아 놓은 겁니다.ㅎㅎ

**Comment by darron1217** [comment]
아아 제가 내용을 제대로 안읽어 봤네요
무슨뜻인지 이해했습니다 :)
